index.html

```diff
@@ -64,98 +92,122 @@ <h2 class="Title-post-1">Multi-Post Stories</h2>
-                            <div class="tags">
-                                <div>
-                                    <ul  class="tag-link">
-                                        <li class="tag "><a href="https://www.html.org">html</a></li>
-                                        <li class="tag "><a href="https://getbootstrap.com/">bootstrap</a></li>
-                                        <li class="tag "><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
-                                    </ul>
+                                <div class="card-7 card card-sec-6 card-fill">
+                                    
+                                        <h2 class="Title-post mob">Profesional Art Printing Data</h2>
+                                        <h2 class="Title-post desk">Website Protfolio </h2>
+                                        <p class="Supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    
+                                    <div class="tags">
+                                        <div>
+                                            <ul  class="tag-link">
+                                                <li class="tag "><a href="https://www.html.org">html</a></li>
+                                                <li class="tag "><a href="https://getbootstrap.com/">bootstrap</a></li>
+                                                <li class="tag "><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+                                            </ul>
+                                        </div>
+                                        <span class="see-project"><a href="">See project</a></span>
+                                    </div>
+                                    
                                 </div>
-                                <span class="see-project"><a href="">See project</a></span>
-                            </div>
-                            
-                        </div>
-                    </div> 
+                            </div> 
                 </section>
                 <section>
                     <div class="About">
-                        <h2 class="about-me">About me</h2>
-                        <span><p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></span>
-                        <span class="get-resume"><a href="">Get My Resume</a></span>
-                        <div class="image-2">
-
+                        
+                            <h2 class="about-me">About me</h2>
```

Comment:
It seems that this button has been misaligned while adding some query. I have tested your design between `760 - 1440` sizes, it's looking the same.

Try to make a design as close to FIGMA design as possible.

| Your Design  | FIGMA Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/213813391-59dc8347-33ca-4575-8ed5-60045d5919a6.png)  | ![image](https://user-images.githubusercontent.com/61361037/213813422-c4fcf81a-00df-4a94-89b5-34ef50b8be7e.png)  |